package.json

```diff
@@ -119,7 +119,8 @@
     "vite-plugin-istanbul": "5.0.0"
   },
   "resolutions": {
-    "immutable": "4.3.2"
+    "immutable": "4.3.2",
+    "i18next": "23.4.4"
```

Comment:
If you update to the latest versions of `@graasp/query-client`, `@graasp/sdk` and `@graasp/ui` you should not need these resolutions anymore.